lib/teamsnap.rb

```diff
@@ -131,15 +131,24 @@ def init(opts = {})
       apply_endpoints(self, collection) && true
     end
 
-    def run(via, href, args = {})
+    def run(via, href, args = {}, opts = {})
       resp = client.send(via, href, args)
 
       if resp.status == 200
-        Oj.load(resp.body)
-          .fetch(:collection)
+        collection = Oj.load(resp.body).fetch(:collection)
+        if opts[:backup_cache]
+          File.open(opts[:backup_cache], "wb") do |f|
+            f.write Marshal.dump(collection)
```

Comment:
Yeah, I'll switch it over to JSON.. It will be more human readable which could be helpful long term